src/terrain/clients/qms/model.clj

```diff
@@ -0,0 +1,246 @@
+(ns terrain.clients.qms.model
```

Comment:
This was a somewhat painful method for generating protocol buffers objects, but [pronto](https://github.com/AppsFlyer/pronto) hasn't been updated to work with newer protocol buffers APIs, and taking the time to update it seemed overwhelming. This works for the time being, but it's not really sustainable. We'll have to find a better option in the near future.